Sources/SecureXPC/Server/XPCMachServer.swift

```diff
@@ -159,9 +163,12 @@ internal class XPCMachServer: XPCServer, NonBlockingStartable {
              xpc_connection_set_event_handler(connection, { event in
                  self.handleEvent(connection: connection, event: event)
              })
+             self.connections.append(WeakConnection(connection))
              xpc_connection_resume(connection)
          })
          xpc_connection_resume(machService)
+        
+        self.machService = machService
```

Comment:
Could be a github thing, but I think the indentation is off here (and a few other places). Could you look it over?

Reply:
The indentation is off, I believe this is the tabs/spaces thing going on. I decided to not change the surrounding lines to make it clearer what the non-whitespace changes actually were.